contracts/silos/NOPSilo.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.10;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+import "contracts/interfaces/ISilo.sol";
+
+contract NOPSilo is ISilo {
+    /// @inheritdoc ISilo
+    string public name;
+
+    constructor(address _token) {
+        name = string(abi.encodePacked(IERC20Metadata(_token).symbol(), " no-op Silo"));
+    }
```

Comment:
Does it need to be assigned a token? As a no-op silo

Reply:
Lol I guess not